styles/pypi/catppuccin.user.css

```diff
@@ -0,0 +1,1080 @@
+
+    body {
+      background-color: @base !important;
+      color: @text !important;
+    }
+
+    input:not([type]),
+    input[type="color"],
+    input[type="date"],
+    input[type="datetime-local"],
+    input[type="datetime"],
+    input[type="email"],
+    input[type="month"],
+    input[type="number"],
+    input[type="password"],
+    input[type="search"],
+    input[type="tel"],
+    input[type="text"],
+    input[type="time"],
+    input[type="url"],
+    input[type="week"],
+    select,
+    textarea {
+      background-color: @mantle !important;
+      border-color: @subtext0 !important;
+      color: @text !important;
+    }
+
+    input:not([type]):active,
+    input:not([type]):focus,
+    input:not([type]):hover,
+    input[type="checkbox"]:active,
+    input[type="checkbox"]:focus,
+    input[type="checkbox"]:hover,
+    input[type="color"]:active,
+    input[type="color"]:focus,
+    input[type="color"]:hover,
+    input[type="date"]:active,
+    input[type="date"]:focus,
+    input[type="date"]:hover,
+    input[type="datetime-local"]:active,
+    input[type="datetime-local"]:focus,
+    input[type="datetime-local"]:hover,
+    input[type="datetime"]:active,
+    input[type="datetime"]:focus,
+    input[type="datetime"]:hover,
+    input[type="email"]:active,
+    input[type="email"]:focus,
+    input[type="email"]:hover,
+    input[type="month"]:active,
+    input[type="month"]:focus,
+    input[type="month"]:hover,
+    input[type="number"]:active,
+    input[type="number"]:focus,
+    input[type="number"]:hover,
+    input[type="password"]:active,
+    input[type="password"]:focus,
+    input[type="password"]:hover,
+    input[type="search"]:active,
+    input[type="search"]:focus,
+    input[type="search"]:hover,
+    input[type="tel"]:active,
+    input[type="tel"]:focus,
+    input[type="tel"]:hover,
+    input[type="text"]:active,
+    input[type="text"]:focus,
+    input[type="text"]:hover,
+    input[type="time"]:active,
+    input[type="time"]:focus,
+    input[type="time"]:hover,
+    input[type="url"]:active,
+    input[type="url"]:focus,
+    input[type="url"]:hover,
+    input[type="week"]:active,
+    input[type="week"]:focus,
+    input[type="week"]:hover,
+    select:active,
+    select:focus,
+    select:hover,
+    textarea:active,
+    textarea:focus,
+    textarea:hover {
+      border-color: @accent-color !important;
+      box-shadow: inset 0 0 0 1px @accent-color !important;
+    }
+
+    input:disabled,
+    input:disabled:active,
+    input:disabled:focus,
+    input:disabled:hover,
+    select:disabled,
+    select:disabled:active,
+    select:disabled:focus,
+    select:disabled:hover {
+      background-color: @mantle !important;
+    }
+
+    hr {
+      background-color: @subtext0 !important;
+      background-image: linear-gradient(90deg, @subtext0, @text) !important;
+    }
+
+    ::-moz-selection {
+      background-color: @accent-color !important;
+      color: @base !important;
+    }
+
```

Comment:
```suggestion
```

I don't believe this is necessary.

Reply:
it technically isn't although would increase compatibility with older firefox versions

Reply:
Not a priority for us. Users should be on the latest version of their respective browsers.